src/main/java/org/bytedeco/javacpp/tools/Parser.java

```diff
@@ -2594,7 +2595,7 @@ boolean function(Context context, DeclarationList declList) throws ParserExcepti
                               +  "_" + dcl.javaName + (dcl.parameters.names == null ? "()" : dcl.parameters.names) + "; }\n";
                     dcl.javaName = "_" + dcl.javaName;
                 }
-                decl.text += modifiers + type.annotations + context.shorten(type.javaName) + " " + dcl.javaName + dcl.parameters.list + ";\n";
+                decl.text += nativeModifiers + type.annotations + context.shorten(type.javaName) + " " + dcl.javaName + dcl.parameters.list + ";\n";
```

Comment:
We don't want to modifier `modifiers` here because it's reused in next iterations of the `for (int n = -2; n < Integer.MAX_VALUE; n++)` loop.

Reply:
Right, ok, that `modifiers` though is also "native". Let's name the copy `privateModifiers` maybe if you want to give it a meaningful name? I'd just go with `modifiers2`, but anyway, `privateModifiers` sounds good.

Reply:
Done